SpatialGDK/Source/SpatialGDKEditorToolbar/Private/SpatialGDKEditorToolbar.cpp

```diff
@@ -892,6 +892,10 @@ void FSpatialGDKEditorToolbarModule::VerifyAndStartDeployment(FString ForceSnaps
 void FSpatialGDKEditorToolbarModule::StartLocalSpatialDeploymentButtonClicked()
 {
 	VerifyAndStartDeployment();
+	// UNR-4334 - Also plug into the new runtime fetching using SpatialPackageManager
```

Comment:
isn't this PR for ticket UNR-4334?

Reply:
Aye - let's change to the tracking epic for this dev branch `UNR-4251`

Reply:
This TODO is redundant? We are already calling
FSpatialPackageManager::TryFetchRuntimeBinary(RuntimeVersion);
here